website/docusaurus.config.js

```diff
@@ -59,6 +60,20 @@ const config = {
       };
     },
   },
+  plugins: [
+    [
+      'client-redirects',
+      /** @type {import('@docusaurus/plugin-client-redirects').Options} */
+      ({
+        redirects: [
+          {
+            from: '/guides',
+            to: '/category/guides',
```

Comment:
let's update the URLs in md files too

Reply:
The `md` files already get `/guides` as the root...I know, it's strange.

Reply:
I think the URL was /guides (as I added the guides) but it looks like it might have changed maybe after a docusaurus update? (or I made a mistake with the URL)